test/e2e/kubernetes/job/job.go

```diff
@@ -0,0 +1,173 @@
+package job
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os/exec"
+	"regexp"
+	"time"
+
+	"github.com/Azure/acs-engine/test/e2e/kubernetes/pod"
+	"github.com/Azure/acs-engine/test/e2e/kubernetes/util"
+)
+
+// List is a container that holds all jobs returned from doing a kubectl get jobs
+type List struct {
+	Jobs []Job `json:"items"`
+}
+
+// Job is used to parse data from kubectl get jobs
+type Job struct {
+	Metadata pod.Metadata `json:"metadata"`
```

Comment:
this pod

Reply:
The types are functionally identical in how we currently use them, so I didn't want to create a duplicate type. `job.Spec`, for example, is different than `pod.Spec`, so I created a distinct `job` type definition for it.